client/extensions/woocommerce/index.js

```diff
@@ -114,17 +103,46 @@ const getStorePages = () => {
 		{
 			container: SettingsPayments,
 			configKey: 'woocommerce/extension-settings-payments',
-			path: '/store/settings/:site/payments',
+			path: '/store/settings/payments/:site',
+			sidebarItem: {
+				isPrimary: false,
+				label: translate( 'Payments' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **43** times:
`translate( 'Payment' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).